torch/csrc/jit/graph_executor.cpp

```diff
@@ -127,7 +127,8 @@ struct ExecutionPlan {
       // Why do I have to care here whether v has a grad_fn or grad accumulator?
       // Why do I have to care here about output_nr? I just want to say
       // grad_fn->setOutputTo(i, v.input_port());
-      grad_fn->next_functions.emplace_back(v.grad_fn() ? v.grad_fn() : v.grad_accumulator(), v.output_nr());
+      // TODO(psag): remove above rant once Function API is improved.
```

Comment:
Well it is already resolved, isn't it?

Reply:
I wanted to wait until I get to give `Function` a nicer API that doesn't require touching the `next_functions` member, i.e. until you can really write `grad_fn->setOutputTo` or something similar.

Reply:
Well I don't think that's necessary. It's very, very, very common to do all kinds of different transformations on `next_functions`, and I don't think `setOutputTo` is particularly helpful for this, without requiring a lot of moves etc.